src/encoded/static/components/app.js

```diff
@@ -70,6 +71,50 @@ const biosampleTypeList = [
     'in vitro differentiated cells',
     'induced pluripotent stem cell line'
 ];
+const assayTermNameList = [
```

Comment:
I suppose I could do a search of experiments and grab the assay_title array from facets. Have to think about when to do this and how to store it.